test/k8sT/Policies.go

```diff
@@ -478,68 +471,59 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 
 		It("Denies traffic with cnp default-deny ingress policy", func() {
 
-			res := kubectl.ExecPodCmd(
```

Comment:
Why was this removed? I see no reason to remove the checking of connectivity before importing policy.

Reply:
We already test in the first test that the connectivity works correctly. In all policy test we make the policy import and we don't test the connectivity. The main reason, is to keep the test as small as possible.